src/store/reducers/capabilities/hooks.ts

```diff
@@ -63,7 +63,7 @@ export const useViewerNodesHandlerHasGroupingBySystemState = () => {
 };
 
 export const useViewerNodesHandlerHasNetworkStats = () => {
-    return useGetFeatureVersion('/viewer/nodes') > 12;
+    return useGetFeatureVersion('/viewer/nodes') > 13;
```

Comment:
Network table is now an experiment, there is no need to implement features inside the experiment with their own feature flags. Peer role filter was added in 14 version, so the whole experiment version will be 13+